packages/grid/scss/_mixins.scss

```
@@ -320,12 +320,9 @@ $carbon--aspect-ratios: ((16, 9), (2, 1), (4, 3), (1, 1), (1, 2));
     @include carbon--make-row();
   }
 
-  .#{$prefix}--grid--condensed .#{$prefix}--row:not(:last-of-type) {
-    margin-bottom: $condensed-gutter;
-  }
-
-  .#{$prefix}--row--condensed + .#{$prefix}--row--condensed {
-    margin-top: $condensed-gutter;
+  .#{$prefix}--grid--condensed [class*='#{$prefix}--col-'] {
```

```suggestion
  .#{$prefix}--grid--condensed [class*='#{$prefix}--col'] {
```

I think it may need to drop the `-` as we do have `bx--col` classes

---

Sorry, looks like I must have missed this while catching up on emails after I was out of office. Valid point in your comment, I made updates based on your suggestion. Let me know if there is anything else.